### DISCUSSION TITLE
Welcome to TFLint Discussions!

### DISCUSSION THREAD

**Discussion by wata727** [terraform-linters/tflint#1418] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wata727** [comment]
Enabled GitHub Discussions. This can help you share your questions, ideas and knowledge. In the future, issues will only deal with bug reports and feature requests that are clear on how to implement them. I don't expect to migrate existing issues, but newly opened issues may be moved to Issues or Discussions as appropriate.

Keep in mind that maintainers have limited time, so questions and ideas are not always answered. We want our users to work together to solve the problem. Have a good day.

**Comment by lukeb12** [comment]
Parallels that never instersect. Threads, and Maps of Meaning

**Reply by lukeb12** [reply]
@Claude @Copilot

**Reply by lukeb12** [reply]
@Gemini
